prow/gcsupload/run_test.go

```diff
@@ -160,12 +160,14 @@ func TestOptions_AssembleTargets(t *testing.T) {
 			spec := &downwardapi.JobSpec{
 				Job:  "job",
 				Type: testCase.jobType,
-				Refs: kube.Refs{
-					Org:  "org",
-					Repo: "repo",
-					Pulls: []kube.Pull{
-						{
-							Number: 1,
+				Refs: []kube.Refs{
```

Comment:
Is this a no-op indentation change? Maybe drop it

Reply:
those are changed from `kube.Refs` to `[]kube.Refs`, across all testcases :-)